src/main/cli/settings.c

```diff
@@ -703,6 +714,10 @@ const clivalue_t valueTable[] = {
 #ifdef USE_SPEKTRUM_BIND
     { "spektrum_sat_bind",          VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { SPEKTRUM_SAT_BIND_DISABLED, SPEKTRUM_SAT_BIND_MAX}, PG_RX_CONFIG, offsetof(rxConfig_t, spektrum_sat_bind) },
     { "spektrum_sat_bind_autoreset",VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_RX_CONFIG, offsetof(rxConfig_t, spektrum_sat_bind_autoreset) },
+#endif
+#ifdef USE_SERIALRX_SRXL2
+    { "srxl2_unit_id",             VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 0xf }, PG_RX_CONFIG, offsetof(rxConfig_t, srxl2_unit_id) },
+    { "srxl2_baud_rate",           VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_SRXL2_BAUD_RATES }, PG_RX_CONFIG, offsetof(rxConfig_t, srxl2_baud_rate) },
```

Comment:
Is there a plan to add support for bit rates different to 115k and 400k? If not, then this should be done as a `TABLE_OFF_ON` setting `srxl2_speed_115k` or `srxl2_speed_slow` or suchlike.
(This is only a temporary solution anyway, as this will be amalgamated into the Unified UART configuration once this is being done.)

Reply:
Currently we dont have "plans" to. The protocol was developed to support the addition of new bit rates though, should there be a need to have faster speed down the line. These new bit rates would be pre-defined however, rather than being defined as an integer, so I dont mind making it a table that would be srxl2_speed_115k, srxl2_speed_400k, etc.

Reply:
Sorry, I just looked at this and realized it was already in a table. Were you implying to just change the "values" from 115k  to srxl2_speed_115k style?

Reply:
If it's only 2 baud rates for now, it should be using the `TABLE_OFF_ON` enum. As I said, this is only temporary, once we redo the UART configuration to allow setting UART parameters by UART instead of by protocol using it, setting the bit rate for SRXL2 will be done by setting the bit rate for the UART it is using.

Reply:
I can change it to TABLE_OFF_ON since it will be temporary anyway. When the change is made to set bit rate by uart rather than protocol, will those bitrate options be an enum with the selection options limited by the protocol, or how will that be handled so only supported bit rates are selectable? Will it also be possible to have default bit rates selected based on protocol selected?

Reply:
Thanks for changing.
I think we will have to look into how we can support protocols for which the available hardware only supports certain speeds in a way that makes it easy for the user to select the correct speed - Spektrum protocols aren't the only ones that are affected by this.
Potentially this will be done by allowing setting of all possible speeds in the firmware / CLI, but restricting the speeds that are selectable in configurator, based on the protocol that has been selected for the UART. This also caters to situations where the 'official' hardware only supports certain speeds, but open source / aftermarket implementations of the same protocol support different speeds.